src/main/resources/fxml/addvault_new_location.fxml

```diff
@@ -29,16 +29,11 @@
 	<children>
 		<Region VBox.vgrow="ALWAYS"/>
 
-		<VBox spacing="6">
+		<VBox fx:id="radioButtonVBox" spacing="6">
 			<Label wrapText="true" text="%addvaultwizard.new.locationInstruction"/>
-			<RadioButton fx:id="iclouddriveRadioButton" toggleGroup="${predefinedLocationToggler}" text="iCloud Drive" visible="${controller.observedLocationPresets.foundIclouddrive}" managed="${controller.observedLocationPresets.foundIclouddrive}"/>
-			<RadioButton fx:id="dropboxRadioButton" toggleGroup="${predefinedLocationToggler}" text="Dropbox" visible="${controller.observedLocationPresets.foundDropbox}" managed="${controller.observedLocationPresets.foundDropbox}"/>
-			<RadioButton fx:id="gdriveRadioButton" toggleGroup="${predefinedLocationToggler}" text="Google Drive" visible="${controller.observedLocationPresets.foundGdrive}" managed="${controller.observedLocationPresets.foundGdrive}"/>
-			<RadioButton fx:id="onedriveRadioButton" toggleGroup="${predefinedLocationToggler}" text="OneDrive" visible="${controller.observedLocationPresets.foundOnedrive}" managed="${controller.observedLocationPresets.foundOnedrive}"/>
-			<RadioButton fx:id="megaRadioButton" toggleGroup="${predefinedLocationToggler}" text="MEGA" visible="${controller.observedLocationPresets.foundMega}" managed="${controller.observedLocationPresets.foundMega}"/>
-			<RadioButton fx:id="pcloudRadioButton" toggleGroup="${predefinedLocationToggler}" text="pCloud" visible="${controller.observedLocationPresets.foundPcloud}" managed="${controller.observedLocationPresets.foundPcloud}"/>
+			<!-- PLACEHOLDER, more radio buttons are added programmatically via controller -->
```

Comment:
what happens when this becomes a _very long_ list? Shouldn't we rather use some scrollable list element here?

Reply:
I wouldn't overengineer this right now. The assumption, that a user does not have more than 10 different cloud provider is quite reasonable. Additionally, we still have plenty of room in the screen:
![grafik](https://github.com/cryptomator/cryptomator/assets/9036915/780f591e-d04b-4b78-b6e1-d00ad3675db7)

Reply:
ok let's accept the risk for now